src/applications/coronavirus-vaccination/components/MaskedSSNWidget.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useCallback } from 'react';
```

Comment:
Leaving the `useCallback` docs here as a breadcrumb:
https://reactjs.org/docs/hooks-reference.html#usecallback